tests/lib/external/snapd-testing-tools/remote/remote.wait-for

```diff
@@ -110,6 +112,24 @@ wait_for_reboot() {
         return
     fi
 
+    if [ -n "$expect_passphrase" ]; then
+        if [ ! -f "$log_file" ]; then
+            echo "remote.wait-for --expect-passphrase without passing an existing --log-file"
+            return 1
+        fi
+        while [ "$attempts" -ge 0 ]; do
```

Comment:
for loop maybe?

Reply:
Actually, it seems that all `remote.wait-for` helpers use `while`, maybe this can be refactored in a followup PR so the script is consistent.